tasks/cleanup.yml

```diff
@@ -1,13 +1,13 @@
 ---
 - name: remove EOLed opsys
-  command: faf releasemod -o "{{item|selectattr("opsys")}}" --opsys-release "{{item|selectattr("release")}}" -s EOL
-  with_items: '{{eol_opsys:|default([], true)|list}}'
+  command: faf releasemod -o "{{ item.opsys }}" --opsys-release "{{ item.release }}" -s EOL
+  loop: "{{eol_opsys|selectattr('opsys')|selectattr('release')|list}}"
```

Comment:
Nitpick: `selectattr()` without a test specified will evaluate the attribute as a boolean (http://jinja.pocoo.org/docs/2.10/templates/#selectattr). `selectattr('foo', 'defined')` would be more correct.

Reply:
```
opsys=''
selectattr('opsys')
```
This skips the `command`.


```
opsys=''
selectattr('opsys', 'defined')
```
This executes the `command` with empty `-o`.

Reply:
Aha, indeed. Then, I suppose, the alternative would be `selectattr('foo', 'string') | selectattr('foo', 'ne', '')`, or something to that effect. Testing the truthiness makes sense to me now, and, I guess, being very strict will not bring benefits.